### ISSUE TITLE
e1l4 - Invisible papers in car

### ISSUE THREAD

**Issue by ChannelDaniila** [Chrissstrahl/hzm-mohaa-coop-mod#38]
When we arrived at the port and we need to show the documents from the car, they are transparent.

**Comment by Chrissstrahl** [comment]
Screenshot ?

**Comment by ChannelDaniila** [comment]
![изображение](https://user-images.githubusercontent.com/75738095/102017999-2539a400-3d73-11eb-9861-9c04d0df8963.png)

**Comment by Chrissstrahl** [comment]
I think this has to do with players being clued and not visible, because they are all at the same spot.
This could be fixed using a script model, but if that is really worth the effort remains questionable.
Unless smithy gets bored as fuck, I don't see this happening. Changing to enhancement.

**Comment by 1337Smithy** [comment]
Indeed, it's caused by invisible players and their invisible view models. What probably can be done is the first player to show papers  gets temporarily made visible, and no one else. Might not be too jarring for others. Also needs to work with the new cam system I've put in place to stop jitter.
